install/kubernetes/helm/istio-remote/templates/sidecar-injector-configmap.yaml

```diff
@@ -170,6 +170,9 @@ data:
             - NET_ADMIN
           runAsGroup: 1337
           {{ "[[ else -]]" }}
+          {{ if and .Values.global.sds.enabled .Values.global.sds.enableTokenMount }}
```

Comment:
Nit: you can move runAsGroup out of the if/else, and remove the if.
We can run as group 1337 in all cases.